scout/server/blueprints/variants/controllers.py

```diff
@@ -1126,11 +1126,11 @@ def update_form_hgnc_symbols(store, case_obj, form):
         # if symbols are numeric HGNC id, translate to current symbols
         for hgnc_symbol in form.hgnc_symbols.data:
             if hgnc_symbol.isdigit():
-                hgnc_gene = store.hgnc_gene(int(hgnc_symbol), genome_build)
-                if hgnc_gene is None:
+                hgnc_gene_caption = store.hgnc_gene_caption(int(hgnc_symbol), genome_build)
+                if hgnc_gene_caption is None:
                     not_found_ids.append(hgnc_symbol)
                 else:
-                    hgnc_symbols.append(hgnc_gene["hgnc_symbol"])
+                    hgnc_symbols.append(hgnc_gene_caption.get("hgnc_symbol", hgnc_symbol))
```

Comment:
Here also, you will default to build 37, so perhaps use the genome_build param?

Reply:
Already do: check L1129. This is just the dict.

Reply:
Ah yes, I misread, never mind!